ToDoList-Redux/Binders/ActionPrism.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  ActionPrism.swift
+//  ToDoList-Redux
+//
+//  Created by Luiz Rodrigo Martins Barbosa on 11.08.20.
+//
+
+import Foundation
+
+// TODO: Use code-generation
+
+// MARK: - PRISM
+
+extension AppAction {
```

Comment:
I used this Xcode Code Snippet to create these extensions:
https://github.com/SwiftRex/SwiftRex/blob/develop/docs/markdown/CodeSnippet/PrismAssociatedValue.codesnippet

But a Sourcery Template is also available if you like it more:
https://github.com/SwiftRex/SwiftRex/blob/develop/docs/markdown/SourceryTemplates/Prism.stencil